lms/djangoapps/verify_student/tests/fake_software_secure.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Fake Software Secure page for use in acceptance tests.
+"""
+from django.core.urlresolvers import reverse
+from django.views.generic.base import View
+from django.views.decorators.csrf import csrf_exempt
+from django.conf import settings
+
+from email.utils import formatdate
+from edxmako.shortcuts import render_to_response
+from verify_student.models import SoftwareSecurePhotoVerification
+
+
+class SoftWareSecureFakeView(View):
+    """
+    Fake SoftwareSecure view for testing different photo verification statuses
+    and email functionality.
+    """
+
+    @csrf_exempt
```

Comment:
Why do we need to disable CSRF here?  The rendered page should have access to the CSRF token.